facial-age-estimator/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@codait/facial-age-estimator",
+  "version": "0.1.0",
+  "description": "Estimate the age of a face.",
```

Comment:
maybe we can make the **description** match that of the https://github.com/IBM/MAX-Facial-Age-Estimator repo description.

Reply:
I'm gonna leave this one open until we add the face detection.

Reply:
works for me